src/mkdocstrings/_internal/inventory.py

```diff
@@ -167,7 +167,20 @@ def parse_sphinx(cls, in_file: BinaryIO, *, domain_filter: Collection[str] = ())
         for _ in range(4):
             in_file.readline()
         lines = zlib.decompress(in_file.read()).splitlines()
-        items = [InventoryItem.parse_sphinx(line.decode("utf8")) for line in lines]
+        items: list[InventoryItem] = []
+        for line in lines:
+            if len(line) == 0 and len(items) == 0:
+                # Skip empty lines at start of inventory section
+                continue
```

Comment:
This bit just allows inventory files without any items to be loaded, which is used in the round trip test; it can be removed without any dramas, but the empty inventory will also need to be removed from the round trip test (but not the Sphinx loading test).